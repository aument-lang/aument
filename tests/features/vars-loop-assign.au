a = 1;
i = 1;
while i < 4 {
    print a, i;
    a = 10 * i;
    i += 1;
}
print a;