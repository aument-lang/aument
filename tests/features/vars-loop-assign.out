int;1
int;1
int;10
int;2
int;20
int;3
int;30